install_config/redeploying_certificates.adoc

```diff
@@ -131,7 +131,7 @@ To run the *_easy-mode.yaml_*  playbook:
 
 ----
 $ ansible-playbook -v -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/certificate_expiry/easy-mode.yaml
+    usr/share/ansible/openshift-ansible/playbooks/openshift-checks/certificate_expiry/easy-mode.yaml
```

Comment:
Missing a "/" in the beginning , /usr/share/ansible/openshift-ansible/playbooks/openshift-checks/certificate_expiry/easy-mode.yaml

Reply:
Whoops! Thanks.